tests/network-tests/src/scenarios/carthagePoA.ts

```diff
@@ -0,0 +1,19 @@
+import bondingSucceedsInPoA from 'src/flows/carthage/bondingSucceedsInPoA'
```

Comment:
change `src/` in import path, to `../` should be  `../flows/carthage/bondingSucceedsInPoA` 

I'm getting `Error: Cannot find module 'src/flows/carthage/bondingSucceedsInPoA'` when trying to run the scenario.

Reply:
Same issues in other scenario.

For the flows change `src/` to `../../`

Although this problem doesn't show up when running `yarn build` and `yarn lint`, it's caused by the way we run the actual scenario with `node -r ts-node/register  src/scenario-name.ts` which is a bit annoying.

Reply:
fixed in 810027b7697f1e024fd708c813d0e187490788e6